charts/firefly/values.yaml

```diff
@@ -207,6 +207,11 @@ core:
     serviceMonitor:
       enabled: false
       scrapeInterval: 10s
+    prometheusRule:
+      enabled: false
+      criticalSeverity: "critical"
+      warningSeverity: "warning"
+      rules: []
```

Comment:
```suggestion
      rules: |
        {{- include "firefly.defaultPrometheusRules" . }}
```

Reply:
Note this must be a string if passed to `tpl`, and we don't need to indent because its handled for us when we template the string within the `PrometheusRule` YAML.

This will allow users to entirely override the rules if they want which is preferred.